package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Converts an array into a Node readable stream",
```

Comment:
Could you also add entries for `files` and `typings`? (can be similar to https://github.com/rubensworks/fetch-sparql-endpoint.js/blob/master/package.json )

Reply:
Should be added now